site/src/components/SignupForm/index.js

```diff
@@ -257,9 +257,8 @@ The Meshery Playground is connected to live Kubernetes cluster(s) and allows use
         <div className='thankyou-box'>
           <h2>Thank you for your interest in Meshery Playground early access program!</h2>
           <p>
-            You are now signed up for the Meshery Playground early access program and your position
-            on the waiting list is confirmed. Please wait patiently for a response from the Meshery
-            team.
+            Your sign-up is confirmed, and you are now on the waiting list. We will notify you
+            on the status of your access via the email address you provided during registration.
           </p>
           <p>
             If you have any questions in the meantime, please send an email to{' '}
```

Comment:
@nwanduka, Could you please also add the link to our Slack channel and discussion forum.  Currently, we are allowing users to mail us only.

Reply:
Oh yes. Thanks for the reminder. I'll do that now.

Reply:
Done